LICENSE-APACHE

```diff
@@ -174,28 +174,3 @@ TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
    of your accepting any such warranty or additional liability.
 
 END OF TERMS AND CONDITIONS
-
-APPENDIX: How to apply the Apache License to your work.
```

Comment:
The appendix was removed in rustc in https://github.com/rust-lang/rust/pull/67734, without explicit buy-in from all contributors. So I think we are safe to do the same in Miri as well.